docs/cases/cases-overview.asciidoc

```diff
@@ -118,9 +118,9 @@ IMPORTANT: Before importing Lens visualizations, Timelines, or alerts into a spa
 [float]
 [[cases-export]]
 === Export a case
-Use the export option to move cases between different Kibana instances. When you export a case, the following data is exported to a newline-delimited JSON (NDJSON) file: case details, user actions, text string comments, case alerts, and lens visualizations.
+Use the export option to move cases between different Kibana instances. When you export a case, the following data is exported to a newline-delimited JSON (`.ndjson`) file: case details, user actions, text string comments, case alerts, and lens visualizations (which are exported as JSON blobs).
 
-NOTE: Exported alerts contain metadata that allow the case to reference the appropriate alerts after they're imported. Lens visualizations are exported as JSON blobs.
+IMPORTANT: Alerts attached to a case are exported as references containing alert metadata. When a <<cases-import, case is imported>>, it uses these references to re-connect to the appropriate alerts. However, alert references only resolve if the correct alert documents are available in the environment. If they're not, alert details will not load when you access them from the imported case. Links to the rule that produced the alert will resolve if the alert is present in the environment.
```

Comment:
> Links to the rule that produced the alert will resolve if the alert is present in the environment

This is not correct. Links to the rule will resolve only if the **rule** is present in the environment, not the alert. Cases saved two references, one for rules and one for alerts. They are autonomous and they did not depend on each other.

Reply:
@cnasikas -- while Cases only saves the rule reference, the `Rule Details` page supports showing 'deleted rules' (in this case 'missing') so long as an alert exists with the Rule's corresponding `id` (7.x) or `rule_id` (8.x). Please see this comment https://github.com/elastic/security-docs/pull/1637#discussion_r817824441 from @nastasha-solomon for details.

Even if the Rule were to be exported and re-imported alongside the Case, it'll have generated a new `id` on import (not `rule_id` which remains static), and so the Rule Details link from within Cases wouldn't find the Rule since it would be referencing the old `id`. I just tested this by creating a Rule, generating Alerts, creating a Case from the Alert, exporting the Rule, deleting the Rule, importing the Rule, and verifying the Rule link within the Case goes to the old ID which shows the `Deleted Rule` as it was recreated from the most recent Alert that was created before it was deleted.

Case
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/156606043-6bc6f691-c58e-4f6e-9494-a4d660fd0ad4.png" />
</p>

Deleted Rule
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/156606212-0fa11de5-621f-4489-932e-12b11ec88c6b.png" />
</p>

Re-imported Rule

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/156606342-96882544-89e5-4068-83b6-3f7f4794dd2f.png" />
</p>

Reply:
Thank you @spong! I wasn't aware that you can view deleted rules if the alert exists.